### PR TITLE
[FIX] ADD minimal Sirius script for osx and linux

### PR DESCRIPTION
**Pull request by oliveralka**
Minimal script for execution of Sirius in the SiriusAdapter 

Tested:
1. Terminal (Sirius & SiriusAdapter)
2. KNIME 
OSX Sierra
Linux (lukas)

I had issues with it on the abinode - but it seemed that the issue was the same for CometAdapter - KNIME was not executing correctly. 
If you have time could you check if it is running for you on linux (if not we can always merge and see).

### CODE REVIEW

File: MacOS/64bit/Sirius/sirius [hunk 1]
```diff
@@ -71,73 +50,8 @@ if [ -n "$JAVA_HOME" ] ; then
...
-        (3) set -- "$args0" "$args1" "$args2" ;;
-        (4) set -- "$args0" "$args1" "$args2" "$args3" ;;
-        (5) set -- "$args0" "$args1" "$args2" "$args3" "$args4" ;;
-        (6) set -- "$args0" "$args1" "$args2" "$args3" "$args4" "$args5" ;;
-        (7) set -- "$args0" "$args1" "$args2" "$args3" "$args4" "$args5" "$args6" ;;
-        (8) set -- "$args0" "$args1" "$args2" "$args3" "$args4" "$args5" "$args6" "$args7" ;;
-        (9) set -- "$args0" "$args1" "$args2" "$args3" "$args4" "$args5" "$args6" "$args7" "$args8" ;;
-    esac
+<<<<<<< HEAD
+>>>>>>> 0bc31899568525f9edfb6d749650488bb96f05a2
```
Comment (lukaszimmermann) [inline MacOS/64bit/Sirius/sirius]: Remainder from merging

File: MacOS/64bit/Sirius/sirius [hunk 2]
```diff
@@ -158,6 +72,6 @@ fi
 
 
 "$JAVACMD" "$@" &
-wait
 
+wait
```
Comment (lukaszimmermann) [inline MacOS/64bit/Sirius/sirius]: That is different from the Linux version. I guess the call here should be same for both OS.

**Review by lukaszimmermann — CHANGES_REQUESTED** [review]

**Comment by oliveralka** [comment]
fix provided

**Review by lukaszimmermann — APPROVED** [review]

**Comment by hroest** [comment]
@oliveralka can you add licence information to the script, e.g. the regular header that we also use for OpenMS?

**Pull request merged**